addons/bus/static/xml/async.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates>
+    <t t-name="bus.systray.AsyncJobSystrayMenu">
+        <li class="o_async_systray_item">
+            <a role="button" href="#" data-toggle="dropdown" data-display="static"
+               t-attf-class="{{ widget.isOpen ? 'show' : '' }} o_async_toggler dropdown-toggle o-no-caret"
+               title="Background tasks">
+                <t t-set="spinner" t-value="widget.aggreatedState()"/>
+                <span t-if="spinner === 'processing'" class="o_async_processing fa fa-spinner fa-spin"/>
+                <span t-elif="spinner === 'pending'" class="o_async_pending fa fa-spinner"/>
+                <span t-elif="spinner === 'finished'" class="o_async_finished fa fa-tasks"/>
+                <t t-set="actionNeededCount" t-value="widget.actionNeededCount()"/>
+                <span t-if="actionNeededCount > 0" t-esc="actionNeededCount" class="o_ready_job_counter badge badge-pill"/>
+            </a>
+            <div t-attf-class="{{ widget.isOpen ? 'show' : '' }} o_async_dropdown dropdown-menu dropdown-menu-right" role="menu">
+
+                <div class="o_async_jobs">
+                    <div class="o_async_jobs_empty" t-if="_.isEmpty(widget.call('async_job', 'getJobs'))">
+                        There is no background task requiring your attention.
+                    </div>
+                    <t t-foreach="Object.values(widget.call('async_job', 'getJobs'))" t-as="job">
+                        <t t-if="job.name">
+                            <t t-call="bus.systray.AsyncJob"/>
+                        </t>
+                    </t>
+                </div>
+            </div>
+        </li>
+    </t>
+
+    <t t-name="bus.systray.AsyncJob">
+        <div t-attf-class="o_async_job o_async_job_{{ job.state }}"
+             t-attf-title="{{ widget.STATE_DATA[job.state].tooltip }}"
+             t-att-data-job-id="job.id">
+            <div class="o_async_job_state">
+                <span t-attf-class="{{ widget.STATE_DATA[job.state].icon }}"/>
+            </div>
+            <div class="o_async_job_name" t-esc="job.name"/>
+        </div>
+    </t>
```

Comment:
Why not just move this inside the template above?

Having it split in another template suggests `template` in JS or explicit call with `QWeb.render()`, but that's not the case here. Split is natural if it was made with OWL, as this would be another component, but I would not do it if it's not OWL yet.

Another reason to not split into another template is due to template using scope of another template, which makes things harder to understand. With OWL components, it's the scope of the component itself, which is usually nicely documented so it's not a problem with them.

Reply:
The initial discussion we had several months ago was about getting the jobqueue ready for OWL. That's why it is splitted